components/BottomNavigation/examples/BottomNavigationBarControllerExampleViewController.swift

```diff
@@ -18,6 +18,44 @@ import MaterialComponentsBeta.MaterialBottomNavigationBeta
 import MaterialComponents.MaterialBottomNavigation_ColorThemer
 import MaterialComponents.MaterialBottomNavigation_TypographyThemer
 
+class BottomNavigationControllerExampleScrollableChildViewController : UICollectionViewController {
+
+  let numberOfItems = 200
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    collectionView.dataSource = self
+    collectionView.register(UICollectionViewCell.self, forCellWithReuseIdentifier: "cell")
+
+    if #available(iOS 11.0, *) {
+      collectionView.contentInsetAdjustmentBehavior = .always
+      collectionView.insetsLayoutMarginsFromSafeArea = true
+    }
+  }
+
+  override func numberOfSections(in collectionView: UICollectionView) -> Int {
+    return 1
+  }
+
+  override func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+    return numberOfItems
+  }
+
+  override func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+    let cellColors: NSArray = {
+      let colors = NSMutableArray()
+    for i in stride(from: 0, to: 1, by: 1.0 / Double(numberOfItems)) {
+      colors.add(UIColor(hue: CGFloat(i), saturation: 1, brightness: 1, alpha: 1))
+      }
+      return colors.copy() as! NSArray
```

Comment:
Why aren't you using Swift arrays instead?

Reply:
Removed.